src/components/ToggleButton/ToggleButton.scss

```diff
@@ -6,3 +6,111 @@
   color: $teal;
   margin: 0 5px;
 }
+
+/* The switch - the box around the slider */
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+}
+
+/* Hide default HTML checkbox */
+.switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+/* The slider */
+@media screen and (prefers-reduced-motion: reduce) {
+  .slider {
+    position: absolute;
+    cursor: pointer;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: $shadow-grey;
+    --webkit-transition: 0.4s;
+    transition: none;
+
+    &.slider-teal {
+      background-color: $teal;
+    }
+  }
+}
+
+.slider {
```

Comment:
You could refactor this to a mixin in order to remove the duplication here and have something like

```scss
@media screen and (prefers-reduced-motion: reduce) {
  .slider {
    @include with-slider(false);
  }
}

.slider {
  @include with-slider(true);
}
```

Reply:
The duplication was caused by stylelint, I believe. I cleaned it up but without using mixins. If you still want me to turn the slider into a mixin I can do that, just let me know!

Check it out here: https://github.com/mixnjuice/frontend/pull/69/commits/9913d051705c3e5053ed90c93d45523e9141632a

Reply:
Nah, that's fine, looks good to me!